### PR TITLE
Add Flutter SDK

### PR DESCRIPTION
**Pull request by christian-rogobete**
- add flutter sdk
- add missing list bullet points

### CODE REVIEW

File: content/docs/software-and-sdks/index.mdx [hunk 1]
```diff
@@ -53,16 +53,21 @@ The Go SDK is split up into a few separate packages, all of which you can find i
...
+* [Source](https://github.com/Synesso/scala-stellar-sdk)
+* [Docs](https://synesso.github.io/scala-stellar-sdk/)
 
 ### Qt/C++
-[Source](https://github.com/bnogalm/StellarQtSDK)
-[Docs](https://github.com/bnogalm/StellarQtSDK/wiki)
+* [Source](https://github.com/bnogalm/StellarQtSDK)
+* [Docs](https://github.com/bnogalm/StellarQtSDK/wiki)
+
+### Flutter
```
Comment (vcarl) [inline content/docs/software-and-sdks/index.mdx:67]:
```suggestion
### Flutter (beta)
```

**Comment by christian-rogobete** [comment]
Hello @vcarl , thank you for your message. Sure, you can edit the PR.

**Comment by vcarl** [comment]
GitHub permissions won't let me write to your fork 🙂 Could you "accept" the suggestion I left?

**Comment by christian-rogobete** [comment]
Done, thank you for adding it.

**Pull request merged**